source/help/settings/theme-colors.rst

```diff
@@ -118,19 +118,20 @@ Monokai theme
 Solarized Dark theme
 ^^^^^^^^^^^^^^^^^^^^
 
-.. image:: ../../images/theme1.PNG
-  :alt: theme1
+.. image:: ../../images/themeSolarizedDark.PNG
```

Comment:
Thanks @magicmonty. Is it intended that the sidebar active color is not similar to the mention highlight color? Looks good otherwise!

![image](https://cloud.githubusercontent.com/assets/14333569/25863715/17b5311c-34aa-11e7-8b66-02e164bba7d2.png)